lib/internal/test_runner/coverage.js

```diff
@@ -495,8 +495,8 @@ function setupCoverage(options) {
     coverageDirectory,
     originalCoverageDirectory,
     cwd,
-    options.coverageExcludeGlobs,
-    options.coverageIncludeGlobs,
+    options.coverageExcludePatterns,
+    options.coverageIncludePatterns,
```

Comment:
IMO the problem with `coverageIncludePatterns` is that it implies that it's the same as `testIncludePatterns`, which this is not. This is including files by glob, whereas the other is including test's by name